src/Parser/Core/Modules/Features/Checklist2/PreparationRule.js

```diff
@@ -58,6 +58,66 @@ class PreparationRule extends React.PureComponent {
       </React.Fragment>
     );
   }
+  renderFlaskRequirements() {
+    const { thresholds } = this.props;
+
+    return (
+      <React.Fragment>
+        <Requirement
+          name="Flask is used"
+          thresholds={thresholds.flaskPresent}
+        />
+      </React.Fragment>
+    );
+  }
+  renderFoodRequirements() {
+    const { thresholds } = this.props;
+
+    return (
+      <React.Fragment>
+        <Requirement
+          name="Food is up"
```

Comment:
I would use consistent language across consumables. "Flask is used" => "Food is used"